istio-operator/istio-operator.spec

```diff
@@ -63,6 +64,10 @@ rm -rf OPERATOR
 mkdir -p OPERATOR/src/github.com/maistra/istio-operator
 tar zxf %{SOURCE0} -C OPERATOR/src/github.com/maistra/istio-operator --strip=1
 
+pushd OPERATOR/src/github.com/maistra/istio-operator
+%patch0 -p1
```

Comment:
Why are we patching it here instead of applying the changes in the repo directly?